Gemfile

```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
+# Read the current Ruby version from the .ruby-version file
+ruby Bundler.read_file(Bundler.root.join('.ruby-version')).strip
+
```

Comment:
Not strictly necessary but should help ensure that we aren't accidentally using the wrong Ruby version when bumping gems (a mistake I made when bumping the gems in this PR!)

Reply:
Oh, interesting, did you find this somewhere?

Reply:
Yeah this was a bit of a rabbit hole. It went something like:
- [Ruby version in gemfile](https://bundler.io/guides/gemfile_ruby.html)
- https://andycroll.com/ruby/read-ruby-version-in-your-gemfile/
- Then when I discovered that that broke `bundle install` outside of the root dir, I discovered `ruby file: '.ruby-version'` exists: https://github.com/rubygems/rubygems/pull/6876 and fix https://github.com/rubygems/rubygems/pull/6892
- Rather than force everyone to update their Bundler to support the feature, I looked up the source and pulled out         [the one liner](https://github.com/ngan/rubygems/blob/master/bundler/lib/bundler/ruby_dsl.rb#L14).